General/Refactorings.md

```diff
@@ -0,0 +1,651 @@
+# Refactoring in Pharo 8.0
+
+## Introduction
+
+Pharo offers many automatic refactoring tools. Therefore, the objective of this tutorial is to present what refactorings are, when to apply them, how to apply them and to show any possible inconvenience before using them.
+
+## What is refactoring?
+
+Martin Fowler defines refactoring as: "... It is a disciplined technique to restructure an existing body of code, altering its internal structure without changing its external behavior." This means that before carrying out any type of refactoring in the code it is important to have complete test coverage to demonstrate that the code behavior does not change during the refactoring process.
```

Comment:
Maybe additionally to citing Martin Fowler, you can cite the book from where this quote comes from.